packages/desktop/electron/lib/aboutPreload.js

```diff
@@ -8,7 +8,7 @@ contextBridge.exposeInMainWorld('about', {
             const aboutData = {
                 appName: productName,
                 version: data.strings.version.replace('{version}', version),
-                iconPath: `./assets/logos/darkmode/${stage}_firefly_logo.svg`,
+                iconPath: './assets/logos/darkmode/logo_iota.svg',
```

Comment:
This logo does not exist. If a new one is going to be added in the future, I would put a `TODO` comment, if not, I would add one that already exists as `logo_iota_full.svg`

Reply:
forgot to push it :pray: